src/test/java/edu/kis/powp/jobs2d/events/DrawLinesMouseListener.java

```diff
@@ -0,0 +1,54 @@
+package edu.kis.powp.jobs2d.events;
+
+import edu.kis.powp.appbase.Application;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+import edu.kis.powp.jobs2d.features.DriverFeature;
+
+import java.awt.*;
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseListener;
+
+public class DrawLinesMouseListener implements MouseListener {
+
+    private final Application application;
+    private final DriverManager driver;
+
+    public DrawLinesMouseListener(Application application) {
+        this.application = application;
+        this.driver = DriverFeature.getDriverManager();
```

Comment:
DriverManager could be injected to avoid DriverFeature dependency (although it doesn't change much in this case).